cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -41,10 +41,8 @@ spec:
               cpu: 100m
               memory: 100Mi
           env: 
-            - name: INFLUXDB_EXTERNAL_URL
-              value: /api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb:api/db/
```

Comment:
@vishh Not sure if i follow. This is an env var that will only affect this pod, correct ?
The api/v1/proxy URL is still available on the apiserver (as per kubectl cluster-info).
The current configuration (pre PR) is asking grafana to go through kube-proxy which will map the connection back to the same pod, correct ? It actually doesn't send the traffic to the api-server, right ?